mlir/test/Dialect/Shape/canonicalize.mlir

```diff
@@ -1,4 +1,5 @@
 // RUN: mlir-opt -split-input-file -allow-unregistered-dialect -canonicalize="test-convergence" %s | FileCheck %s
+// RUN: mlir-opt -split-input-file -allow-unregistered-dialect -canonicalize="test-convergence top-down=1" %s | FileCheck %s
```

Comment:
In the original issue, we use `top-down=0` to reproduce the case. Is it okay to use `top-down=1` instead?

https://github.com/llvm/llvm-project/pull/118941

Reply:
It should be `top-down=0`. Thanks for your reminder.